BUILD

```diff
@@ -50,6 +50,20 @@ cc_library(
     ],
 )
 
+cc_library(
+    name = "experimental-pubsublite",
+    deps = [
+        "//google/cloud/pubsublite:google_cloud_cpp_pubsublite",
+    ],
+)
+
+cc_library(
+    name = "experimental-pubsublite-mocks",
```

Comment:
nit: the names will change so not a big deal... but seems like we prefer '_' to '-'

Reply:
This is my fault, the `experimental-` prefix is with a dash, the regular rules are with `_`.  Made the prefix consistent. We can fix all the `experimental-` vs. `experimental_` in a separate PR.